 vcl: fix build with vlc backend enabled
 
 avmedia/source/vlc/vlcplayer.cxx tries to access the aWindow member
 directly, which leads to a compilation failure, since it's private.

diff --git include/vcl/sysdata.hxx include/vcl/sysdata.hxx
index 63b7c810de3d..0cd0e037c823 100644
--- a/include/vcl/sysdata.hxx
+++ b/include/vcl/sysdata.hxx
@@ -73,9 +73,8 @@ struct VCL_DLLPUBLIC SystemEnvData
     // however, the GTK3 vclplug wants to store pointers in here!
     sal_IntPtr          aShellWindow;   // the window of the frame's shell
     Platform            platform;       // the windowing system in use
-private:
-    sal_uIntPtr         aWindow;        // the window of the object
 public:
+    sal_uIntPtr         aWindow;        // the window of the object
 
     void SetWindowHandle(sal_uIntPtr nWindow)
     {
